class-genesis-simple-edits.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * The main class that handles the entire output, content filters, etc., for this plugin.
+ *
+ * @package genesis-simple-edits
+ * @since 1.0
+ */
+
+/**
+ * Main Class
+ *
+ * @since 1.0
+ */
+class Genesis_Simple_Edits {
+
+	/**
+	 * Plugin version
+	 *
+	 * @var $plugin_version Plugin Version
+	 */
+	public $plugin_version = '2.2.1';
```

Comment:
We could define a constant in the main plugin file, like `GENESIS_SIMPLE_EDITS_VERSION` and use here as well.